src/battle_util.c

```diff
@@ -6999,6 +7102,9 @@ static u32 CalcAttackStat(u16 move, u8 battlerAtk, u8 battlerDef, u8 moveType, b
             if (gBattleMons[BATTLE_PARTNER(battlerAtk)].species == SPECIES_CHERRIM && IS_MOVE_PHYSICAL(move))
                 MulModifier(&modifier, UQ_4_12(1.5));
             break;
+        case ABILITY_POWER_SPOT:
```

Comment:
according to bulbapedia, this boosts power not attack, in which case it should go under `Battery` in `CalcMoveBasePowerAfterModifiers`

Reply:
Done :eyes: